src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java

```diff
@@ -123,16 +140,18 @@ protected void processFiltered(File file, FileText fileText) {
     }
 
     /**
-     * Sets the maximum length of a line.
+     * Setter to specify the maximum line length allowed.
+     *
      * @param length the maximum length of a line
      */
     public void setMax(int length) {
         max = length;
     }
 
     /**
-     * Set the ignore pattern.
-     * @param pattern a pattern.
+     * Setter to specify pattern for lines to ignore.
+     *
+     *  @param pattern a pattern.
```

Comment:
Double space.
@pbludov Sounds like another Javadoc check you can suggest. :)

Reply:
done